spec/knife_cookbook_doc/attribute_model_spec.rb

```diff
@@ -22,12 +30,25 @@
 =end
 default['knife_cookbook_doc']['attr3'] = 'attr3_value'
 
+=begin
+# <
+Another multiline begin/end with single line of text
+# >
+=end
+default['knife_cookbook_doc']['attr3x'] = 'attr3x_value'
+
 #<
 # Multiline comment with
-multiple lines of text
+# multiple lines of text
```

Comment:
BTW: I corrected this unrelated test content. It passes the test either way. This is the correct ruby, however.